VariablesHomework/VariablesHomework.cs

```diff
@@ -60,86 +77,86 @@ public void Operators()
-            Assert.IsTrue(h == /*Answer here*/); // Make this pass
+            Assert.IsTrue(h == 23); // Make this pass
 
             string l = "Hello! ";
             string m = l + "You!";
-            Assert.IsTrue(m == /*Answer here*/); // Make this pass
+            Assert.IsTrue(m == "Hello! You!"); // Make this pass
 
             int v = 5;
             int x = v % 4;
-            Assert.IsTrue(x == /*Answer here*/); // Make this pass
+            Assert.IsTrue(x == 1); // Make this pass
 
             int finalInt = 15;
             int divisionResult = 15 / 3;
-            Assert.IsTrue(divisionResult == /*Answer here*/); // Make this pass
+            Assert.IsTrue(divisionResult == 5); // Make this pass
 
             finalInt = 15;
             finalInt /= 3;
-            Assert.IsTrue(finalInt == /*Answer here*/); // Make this pass
+            Assert.IsTrue(finalInt == 5); // Make this pass
 
             int orderOfOperations = 2;
             orderOfOperations = 2 + 12 * (orderOfOperations * 3);
-            Assert.IsTrue(orderOfOperations == /*Answer here*/); // Make this pass
+            Assert.IsTrue(orderOfOperations == 74); // Make this pass
         }
 
         [TestMethod]
         public void Ranges()
         {
-            Assert.IsTrue(int.MaxValue == /*Answer here*/); // Make this pass
-            Assert.IsTrue(int.MinValue == /*Answer here*/); // Make this pass
-            Assert.IsTrue(float.MaxValue == /*Answer here*/); // Make this pass
-            Assert.IsTrue(float.MinValue == /*Answer here*/); // Make this pass
-            Assert.IsTrue(double.MaxValue == /*Answer here*/); // Make this pass
-            Assert.IsTrue(double.MinValue == /*Answer here*/); // Make this pass
+            Assert.IsTrue(int.MaxValue == 2147483647); // Make this pass
+            Assert.IsTrue(int.MinValue == -2147483647); // Make this pass
+            Assert.IsTrue(float.MaxValue == 13.5); // Make this pass
+            Assert.IsTrue(float.MinValue == -13.5); // Make this pass
+            Assert.IsTrue(double.MaxValue == 1337.1337); // Make this pass
+            Assert.IsTrue(double.MinValue == -1600.1337); // Make this pass
         }
 
         [TestMethod]
         public void Overflow()
         {
             unchecked // ignore this, it simple means im telling the compiler i know what I am doing.
             {
-                Assert.IsTrue(int.MaxValue + 1 == /*Answer here*/); // Make this pass
-                Assert.IsTrue(int.MinValue - 1 == /*Answer here*/); // Make this pass
-                Assert.IsTrue(float.MaxValue + 1 == /*Answer here*/); // Make this pass
-                Assert.IsTrue(float.MinValue - 1 == /*Answer here*/); // Make this pass
-                Assert.IsTrue(double.MaxValue + 1 == /*Answer here*/); // Make this pass
-                Assert.IsTrue(double.MinValue - 1 == /*Answer here*/); // Make this pass
+                Assert.IsTrue(int.MaxValue + 1 == 2147483648); // Make this pass
```

Comment:
We haven't yet covered what happens when you "overflow" outside of the range of a data type.